Automattic/jetpack
# Boost: Remove global state from Score component

## Proposed changes:

* Moves global store to parent component in favor of props.

### Other information:


## Jetpack product discussion



pc9hqz-286-p2

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Make sure Score section is working as expected (toggling a feature on or off should refresh the scores).
![CleanShot 2023-09-15 at 11 55 12](https://github.com/Automattic/jetpack/assets/11799079/a03e2972-aea7-448a-aa44-2bbcb799ecab)
